adapter/internal/oasparser/envoyconf/http_filters.go

```diff
@@ -44,21 +44,21 @@ import (
 
 // getHTTPFilters generates httpFilter configuration
 func getHTTPFilters() []*hcmv3.HttpFilter {
-	extAauth := getExtAuthzHTTPFilter()
-	router := getRouterHTTPFilter()
-	lua := getLuaFilter()
+
 	cors := &hcmv3.HttpFilter{
 		Name:       wellknown.CORS,
 		ConfigType: &hcmv3.HttpFilter_TypedConfig{},
 	}
+	extAauth := getExtAuthzHTTPFilter()
+	lua := getLuaFilter()
+	router := getRouterHTTPFilter()
 
-	httpFilters := []*hcmv3.HttpFilter{
-		cors,
-		extAauth,
-		lua,
-		router,
+	logConf := config.ReadLogConfigs()
+
+	if logConf.DebugLogs.Enable {
+		return []*hcmv3.HttpFilter{cors, extAauth, getDebugLogFirstFilter(), lua, getDebugLogSecondFilter(), router}
```

Comment:
just a query; if somebody wants to get debug logs before and after interceptor, isn't it enough to enable trace logs? We have added adequate logs in interceptor lua scripts, haven't we.

Reply:
why do we have to forcibly add debug logs, doesn't envoy provide adequate logs (just felt it's highly unlikely)?

Reply:
Envoy doesn't log the message body. In here we are looking for a log that prints headers, body, etc. Somewhat similar to wire logs.